packages/firestore/test/unit/specs/recovery_spec.test.ts

```diff
@@ -75,5 +75,61 @@ describeSpec(
           .expectEvents(query, {});
       }
     );
+
+    specTest('Recovers when write cannot be persisted', [], () => {
```

Comment:
Do these tests need a `no-ios` and `no-android` tag or some new kind of tag to indicate that failure recovery is required?

Reply:
All tests in this file are marked `no-ios` and `no-android` at the moment: https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/test/unit/specs/recovery_spec.test.ts#L26

Reply:
FYI: I pushed a commit to this PR to remove the `durable-persistence` tags as the new recovery tests also work with memory persistence. Most of my changes in my next PR were due to the formatting caused by this change.